clogMonitor/src/components/TimeRange.js

```diff
@@ -40,7 +110,7 @@ const TimeRange = ({ startTime, startChangeHandler, endTime, endChangeHandler, d
     return (
         <FormControl margin="normal" id="timerangeformcontrol" className="timerange">
             <Stack id="timerangestack" spacing={2} direction={direction}>
-                <FormControl id="startformcontrol" error={isInputError(startTime) || isRangeError()}>
+                <FormControl id="startformcontrol" error={isInputError(startTime) || isRangeError() || isDST(startTime)}>
```

Comment:
Looks like isDST(time) just checks for if there is DST conflict, but when there is conflict AND the selection of BEFORE/AFTER has been made, is there still a conflict? Once the user has selected a choice, there shouldn't be an error anymore.